WordPressVIPMinimum/Sniffs/Functions/RestrictedFunctionsSniff.php

```diff
@@ -222,14 +222,12 @@ public function getGroups() {
 			],
 			'file_ops' => [
 				'type'      => 'error',
-				'message'   => 'Filesystem writes are forbidden, you should not be using %s().',
+				'message'   => 'Filesystem writes are forbidden, please do not use %s().',
 				'functions' => [
 					'delete',
-					'file_put_contents',
```

Comment:
I think that at least a comment needs to go into these two removals, explaining why this would appear to be an incomplete list, compared to a docs page.